src/layouts/Resources.jsx

```diff
@@ -58,7 +82,7 @@ export default class Resources extends Component {
     super(props);
     this.state = {
       resourceCategories: [],
-      resourcePages: [],
+      resourcePages: undefined,
```

Comment:
Would it be better to initialize `resourcePages` as `[]` for consistency? Since in our other files we don't initialize with `undefined`. Then, instead of using `resourcePages.length > 0` we can test for whether the array is empty with lodash `_.isEmpty`

Reply:
If we initialize resourcePages as [], and there are no existing resourcePages, the page will display “Loading resources” since it will be empty